staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -266,8 +266,8 @@ func TestSortMergeLists(t *testing.T) {
 	}
 
 	for _, c := range tc.TestCases {
-		original := testObjectToJSONOrFail(t, c.Original, c.Description)
-		sorted := testObjectToJSONOrFail(t, c.Sorted, c.Description)
+		original := sortJsonOrFail(t, testObjectToJSONOrFail(t, c.Original), c.Description)
+		sorted := testObjectToJSONOrFail(t, c.Sorted)
```

Comment:
nit: rename `original` and `sorted` to `got` and `epxected`? It's really up to you :)